schema/resoruce_test.go

```diff
@@ -0,0 +1,7 @@
+package schema
+
+import "testing"
+
+func TestResource(t *testing.T) {
```

Comment:
Empty test file - let's either put some assertions or delete the file so it doesn't look like there is test coverage for resource.go?